CONTRIBUTING.md

```diff
@@ -163,6 +163,10 @@ directory. The proper directory will be full of .gem files.
 #### Getting the data dumps
 * You can use rubygems.org data [dumps](https://rubygems.org/pages/data) to test
 application in development environment especially for performance related issues.
+To use redis data dumps, you should check redis configuration file to make sure
+configuration of the filename where to dump the db and it's directory point to the
+location of the redis dump file you have just downloaded from rubygems.org
+[(See more detail about redis data import)](http://stackoverflow.com/questions/14497234/how-to-recover-redis-data-from-snapshotrdb-file-copied-from-another-machine)
```

Comment:
@arthurnn I wrote it to emphasize this specific part of redis.conf file.
```shell
# The filename where to dump the DB
dbfilename mydump.rdb

# The working directory.
#
# The DB will be written inside this directory, with the filename specified
# above using the 'dbfilename' configuration directive.
#
# The Append Only File will also be created inside this directory.
#
# Note that you must specify a directory here, not a file name.
dir /usr/local/var/db/redis/
```
We can change this part like "You should check redis.conf file to make sure what should be the redis dump filename and where it should be"